src/repl.jl

```diff
@@ -0,0 +1,23 @@
+using ParserCombinator
+using REPL: REPL, LineEdit
+using ReplMaker
+
+function lisp_reader(s)
+    try
+        read(String(take!(copy(LineEdit.buffer(s)))))
+        true
+    catch err
+        isa(err, ParserCombinator.ParserException) || rethrow(err)
+        false
+    end
+end
+
+function init_repl()
+    ReplMaker.initrepl(lisp_eval_helper,
+                  repl = Base.active_repl,
+                  valid_input_checker = lisp_reader,
+                  prompt_text = "jλ> ",
+                  prompt_color = :yellow,
+                  start_key = ")",
+                  mode_name = "Lisp Mode")
+end
```

Comment:
I would instead do this:
```julia
function init_repl(; prompt_text="jλ> ", prompt_color = :red, start_key = ")", sticky=true)
	    ReplMaker.initrepl(lisp_eval_helper,
	                  repl = Base.active_repl,
	                  valid_input_checker = lisp_reader,
	                  prompt_text = prompt_text,
	                  prompt_color = prompt_color,
	                  start_key = start_key,
                          sticky_mode=sticky,
	                  mode_name = "Lisp Mode")
end
```
so that the user may choose the prompt text, colour, key and whether or not the mode is sticky at startup.

Reply:
_Nice_; haven't done Julia since 2015 or so -- _love_ the feedback!